browser-test/initialize_data.py

```diff
@@ -101,9 +99,6 @@ def load_test_data():
         username=data['email'],
         password=data['password'],
         email=data['email'],
-        password_updated=now().date()-timedelta(days=180)
-    )
-    OrganizationUser.objects.create(
-        organization=org,
-        user=expired_password_user
+        password_updated=now().date() - timedelta(days=180),
     )
+    OrganizationUser.objects.create(organization=org, user=expired_password_user)
```

Comment:
These code changes are not relevant in this docs PR, please revert these, focus each PR on one goal.

Reply:
@nemesifier , I ran ```openwisp-qa-format``` and it did this. But I will revert before pushing, np.